### PR TITLE
bug fix

### PR DESCRIPTION
**Pull request by Ellenn-A**
# Changes 
- updated errors on offer-credential endpoint 
- version bump

### CODE REVIEW

File: src/controllers/credentials/CredentialController.ts [hunk 1]
```diff
@@ -198,12 +199,17 @@ export class CredentialController extends Controller {
     @Res() internalServerError: TsoaResponse<500, { message: string }>
   ) {
     try {
+      //check connection exists
+      const connection = await this.agent.connections.findById(options.connectionId)
+      if (!connection)
+        return notFoundError(404, { reason: `connection with connection id "${options.connectionId}" not found.` })
```
Comment (jonmattgray) [inline src/controllers/credentials/CredentialController.ts:205]: add a test for this `404`?

File: src/controllers/credentials/CredentialController.ts [hunk 2]
```diff
@@ -14,6 +14,7 @@ import {
   AcceptCredentialOfferOptions,
   CreateOfferOptions,
 } from '../types'
+import { option } from 'yargs'
```
Comment (jonmattgray) [inline src/controllers/credentials/CredentialController.ts:17]: to delete

File: src/controllers/credentials/CredentialController.ts (hunk 1 above)
Comment (Ellenn-A) [reply src/controllers/credentials/CredentialController.ts:204]: So I did I just don't know how to extract the error message from the response - but idk if we need that. The error response looks like this: ` @Res() notFoundError: TsoaResponse<404, { reason: string }>,`

Comment (jonmattgray) [reply src/controllers/credentials/CredentialController.ts:204]: I think you'd look at `response.body` but you'd have to be quite specific to match the `reason`, seems fine as is to me

**Review by jonmattgray — APPROVED** [review]

**Pull request merged**